packaging/dbscripts/upgrade/pre_upgrade/0000_config.sql

```diff
@@ -1113,9 +1113,8 @@ select fn_db_update_config_value('VdcVersion','4.4.0.0','general');
 -- ProductRPMVersion is automatically updated by engine-setup,
 -- but it doesn't harm to keep it up to date here as well.
 select fn_db_update_config_value('ProductRPMVersion','4.4.0.0','general');
-select fn_db_update_config_value_for_versions_from_up_to('VdsFenceOptionMapping','apc:secure=secure,port=ipport,slot=port;apc_snmp:port=port,encrypt_options=encrypt_options;bladecenter:secure=secure,port=ipport,slot=port;cisco_ucs:secure=ssl,slot=port;drac5:secure=secure,slot=port;drac7:;eps:slot=port;hpblade:port=port;ilo:secure=ssl,port=ipport;ipmilan:;ilo2:secure=ssl,port=ipport;ilo3:;ilo4:;ilo_ssh:port=port;redfish:port=ipport,secure=ssl;rsa:secure=secure,port=ipport;rsb:;wti:secure=secure,port=ipport,slot=port', '4.2', '4.3');
```

Comment:
it would most probably be alright to make this change at this point but it can theoretically be problematic if:
1. we changed `fn_db_add_config_value_for_versions_from_up_to('VdsFenceOptionMapping' ...` after it was introduced
2. we upgrade a 4.2 or 4.3 environment that doesn't hold the latest value of VdsFenceOptionMapping

how about not dropping the 4.2 and 4.3 updates?